resolwe_bio/processes/workflows/geo_import.py

```diff
@@ -0,0 +1,343 @@
+        "growth_protocol_ch1": "growth_protocol",
+        "treatment_protocol_ch1": "treatment_protocol",
+    }
+    if (
+        "organism_ch1" in metadata.columns
+        and sample_metadata["organism_ch1"] in species
+    ):
+        descriptor["general"]["species"] = sample_metadata["organism_ch1"]
+
+    if "cell line" in metadata.columns:
+        descriptor["general"]["biosample_type"] = "cell_line"
+        descriptor["general"]["cell_line"] = sample_metadata["cell line"]
+    elif "tissue" in metadata.columns:
+        descriptor["general"]["biosample_type"] = "tissue"
+    if "contact_name" in metadata.columns:
+        descriptor["general"]["annotator"] = sample_metadata["contact_name"].replace(
+            ",,", " "
+        )
+
+    for geo_attribute, attribute in general_attributes.items():
+        if geo_attribute in metadata.columns:
+            descriptor["general"][attribute] = sample_metadata[geo_attribute]
+
+    if "library_strategy" in metadata.columns:
+        formated_assay = sample_metadata["library_strategy"].lower().replace(" ", "-")
+        if formated_assay in assay_types:
+            descriptor["experiment"]["assay_type"] = formated_assay
+    if "extract_protocol_ch1" in metadata.columns:
+        descriptor["experiment"]["extract_protocol"] = sample_metadata[
+            "extract_protocol_ch1"
+        ]
+    if "molecule_ch1" in metadata.columns:
+        formated_molecule = sample_metadata["molecule_ch1"].lower().replace(" ", "_")
+        if formated_molecule in molecule_choices:
+            descriptor["experiment"]["molecule"] = formated_molecule
+    if "instrument_model" in metadata.columns:
+        formated_platform = (
+            sample_metadata["instrument_model"]
+            .replace("Illumina ", "")
+            .lower()
+            .replace(" ", "_")
+        )
+        if formated_platform in platform_types:
+            descriptor["experiment"]["platform"] = formated_platform
+    return descriptor
+
+
+class GeoImport(Process):
+    """Import all runs from a GEO Series.
+
+    WARNING: Additional costs for storage and processing may be incurred
+    if a very large data set is selected.
+
+    This runs SRA import process for each individual experiment (SRX)
+    from the selected GEO Series. In addition metadata table with sample
+    information is created and uploadad to the same collection.
+    Currently only RNA-seq datasets are supported.
+    """
+
+    slug = "geo-import"
+    name = "GEO import"
+    requirements = {
+        "expression-engine": "jinja",
+        "executor": {
+            "docker": {
+                "image": "public.ecr.aws/s4q6j6e8/resolwebio/common:2.7.0",
+            },
+        },
+        "resources": {
+            "cores": 1,
+            "memory": 8192,
+            "network": True,
+        },
+    }
+    data_name = "{{ gse_accession }}"
+    version = "1.0.0"
+    process_type = "data:geo"
+    category = "Pipeline"
```

Comment:
Related SRA import processes are in `category = "Import"`. Add also this one to the same category, or create a new `SRA data` category

Reply:
Good suggestion, I will add it to the `Import` category to keep it consistent with SRA import.